ush/set_fv3nml_sfc_climo_filenames.py

```diff
@@ -1,33 +1,42 @@
 #!/usr/bin/env python3
 
+"""
+Update filenames for surface climotology files in the namelist.
+"""
+
+import argparse
 import os
+import re
 import sys
-import argparse
 from textwrap import dedent
 
+from uwtools.api.config import get_yaml_config, realize
+
 from python_utils import (
-    print_input_args,
-    print_info_msg,
-    print_err_msg_exit,
+    cfg_to_yaml_str,
     check_var_valid_value,
-    mv_vrfy,
-    mkdir_vrfy,
-    cp_vrfy,
-    rm_vrfy,
+    flatten_dict,
     import_vars,
-    set_env_var,
-    load_config_file,
     load_shell_config,
-    flatten_dict,
-    define_macos_utilities,
-    find_pattern_in_str,
-    cfg_to_yaml_str,
+    print_info_msg,
 )
 
-from set_namelist import set_namelist
+VERBOSE = os.environ.get("VERBOSE", "true")
+
+NEEDED_VARS = [
+    "CRES",
+    "DO_ENSEMBLE",
+    "EXPTDIR",
+    "FIXlam",
+    "FV3_NML_FP",
+    "PARMdir",
+    "RUN_ENVIR",
+    ]
```

Comment:
Here, I'm limiting the namespace of global variables this script requires from all the ones in the general sections of the experiment config file.